app/chronology/models.py

```diff
@@ -6,9 +6,12 @@
 
 
 class Festival(models.Model):
+    def default_festival_settings():
+        return {"theme": {}, "forms": {}}
+
     end_time = models.DateTimeField()
     name = models.CharField(max_length=255)
-    settings = JSONField(null=True)
+    settings = JSONField(default=default_festival_settings)
```

Comment:
shouldn't it have `self` reference?

Reply:
It's a function in class' scope, not a method.
Anyway, I got it out of class because pylint